pkg/istio/common/common.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021, Verizon Media Inc.
+// Licensed under the terms of the 3-Clause BSD license. See LICENSE file in
+// github.com/yahoo/k8s-athenz-istio-auth for terms.
+package common
```

Comment:
Can we combine this file with the util.go file? One file with the common functionality should be enough.